src/screens/Profile/Notification.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { Card, Button, Paragraph } from 'react-native-paper';
+import Swipeable from 'react-native-gesture-handler/Swipeable';
+import PropTypes from 'prop-types';
+
+export default class Notification extends React.Component {
+  constructor(props) {
+    super(props);
+    const { postTitle, replies } = this.props;
+    this.onClearPress = this.onClearPress.bind(this);
+    this.onViewPress = this.onViewPress.bind(this);
+    this.notificationDescriptions = {
+      reply: `Your post '${postTitle}' has recieved ${replies} replies. Click to go to post.`,
+      approve: `Your post '${postTitle}' has been approved. Click to go to post.`,
+    };
+  }
+
+  onViewPress() {
+    const { notificationID, viewPress } = this.props;
+    viewPress(notificationID);
+  }
+
+  async onClearPress() {
+    const { notificationID, clearPress } = this.props;
+    clearPress(notificationID);
+  }
+
+  render() {
+    const { type, viewed } = this.props;
+    const viewStatus = viewed ? notificationStyles.inactiveNotification
+      : notificationStyles.activeNotification;
+    let text;
+    if (type === 'reply') {
+      text = this.notificationDescriptions.reply;
+    } else if (type === 'approve') {
+      text = this.notificationDescriptions.approve;
+    }
```

Comment:
This can be written in one line without conditionals: `const notificationDescription = notificationDescriptions[type];`